src/pages/drive/addIns/driveAddIns.tsx

```diff
@@ -0,0 +1,529 @@
+        <li><p><strong>onShutdown:</strong> The onShutdown property must be set to true to execute an Add-In when logging out of Geotab Drive.</p></li>
+    </ol>
+    <InformationalBox>
+        <p>More information about these 2 methods can be found in this {" "}
+            <a href="https://docs.google.com/document/d/1-r9o9epj61WMmGxRveA9SXR86lQGHcxgMh8lsVXGL54/edit?usp=sharing" target="_blank"
+                rel="noopener noreferrer"
+                aria-label="">document</a>
+            .</p>
+    </InformationalBox>
+
+    <p>e.g.</p>
+    <CodeSample
+        language="json"
+        code={`{
+    "name": "addin-name",
+    "supportEmail": "mysupport@support.com",
+    "version": "1.0.0",
+    "items": [{
+        "url": "index.html",
+        "path": "DriveAppLink/",
+        "menuName": {
+            "en": "Add-in"
+        },
+        "icon": "images/icon.svg"
+    }, {
+        "url": "myGeotabConfigurationPage.html",
+        "path": "AdministrationLink/",
+        "menuName": {
+            "en": "Add-in Configurations"
+        },
+        "icon": "images/icon.svg"
+    }],
+    "onStartup": true,
+    "onShutdown": true
+}`}></CodeSample >
+</div >;
+
+
+const driveLifeCycle: ReactNode = <div className="paragraph">
+    <p>The onStartup and onShutdown properties respectively enable the <strong>startup</strong> and <strong>shutdown</strong> lifecycle methods.</p>
+    <h2>Startup</h2>
+    <p>When the dashboard page is visible, the startup method is only called once. If the user navigates away from the page then navigates back, the startup method is not called again. If the Add-In requires re-initialization, the user must either log out and log in again, or refresh the application.</p>
+    <CodeSample
+        language="javascript"
+        code={`startup: function (freshApi, freshState, initializeCallback) {
+    // Code that needs to be executed on dashboard should go here
+    initializeCallback();
+}`}></CodeSample >
+    <h2>Shutdown</h2>
+    <p>Shutdown Add-Ins are executed when the final driver logs out of the Drive App. If there are co-drivers, and one of the co-drivers logs out (while other drivers remain logged in to the Drive App), the shutdown Add-In is not executed.</p>
+    <p>Additionally, the Add-In is expected to return a promise since shutdown Add-Ins have a 15-second time limit to perform their function before the Add-Ins time out and the logout process is completed. The time limit prevents the application from freezing in the middle of the logout process as a result of faulty Add-Ins.</p>
+    <CodeSample
+        language="javascript"
+        code={`shutdown: function (api, state, callback) {
+    return new Promise(resolve => {
+        // Do work, make any api calls etc
+        resolve() // eventually need to call this somewhere so the promise resolves
+    });
+}`}></CodeSample >
```

Comment:
```suggestion
}`}/>
```
This can be self closing cause it has no children inside of it.